swan/templates/storageclasses.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if .Values.cvmfsClient.csiDriver }}
+{{ if or .Values.cvmfs.deployCsiDriver .Values.cvmfs.useCsiDriver }}
 {{- range .Values.cvmfs.repositories }}
```

Comment:
This needs fixing, right? It's no longer in `cvmfs.repositories` but `singleuser.cvmfs.repositories` right?

Reply:
Edited your last commit to fix it. Shout if I'm wrong @ebocchi